xmipp

```diff
@@ -1038,17 +1127,32 @@ def compile_cuFFTAdvisor():
 
 
 def compile(Nproc):
+    if isinstance(Nproc, str) and Nproc.startswith('N='):
+        Nproc = int(Nproc.split('N=')[1])
     ensureConfig()
     ensureConfigVersion(configDict)
     if not compileDependencies(Nproc):
         return False
     return compileXmipp(Nproc)
 
 
+def installDepConda(dep):
+    condaEnv = os.environ.get('CONDA_DEFAULT_ENV', 'base')
+    if condaEnv != 'base' and runJob("conda activate %s ; "
+                                     "conda install %s -y"
+                                     % (condaEnv, dep)):
+        print(green("'%s' installed using conda" % dep))
+        return True
+    return False
+
+
 def compileDependencies(Nproc):
     print("Building Dependencies -------------------------------------")
     result = True
     if is_config_true('CUDA'):
+        if not checkProgram('nvcc', False) and checkProgram(configDict['NVCC'], False):
```

Comment:
use only what you have in the config file

Reply:
I refactored this in order to make it clear.

The point is, if the basename of the cuda-compiler is not found but the whole path yes, then the dirname of that compiler must be added to the PATH in order to compile cuFFTAdvisor.

Reply:
What I'm trying to say is, that if we have some settings in the config file, we should use only whatever is there